multik-api/src/commonMain/kotlin/org/jetbrains/kotlinx/multik/ndarray/operations/Transformation.kt

```diff
@@ -88,9 +89,20 @@ public fun <T, D : Dimension> MultiArray<T, D>.repeat(n: Int): D1Array<T> {
     val data = initMemoryView<T>(size * n, dtype)
     if (consistent) {
         when (this.dtype) {
+            DataType.ByteDataType -> this.data.getByteArray()
```

Comment:
Probably need to write copyInto for the MemoryView. I will create an issue on this.